src/app/contextBroker/contextBroker.cpp

```diff
@@ -246,6 +247,7 @@ int             writeConcern;
 #define HTTPSCERTFILE_DESC  "certificate key file (for https)"
 #define RUSH_DESC           "rush host (IP:port)"
 #define MULTISERVICE_DESC   "service multi tenancy mode"
+#define ALLOWED_ORIGIN_DESC "CORS allowed origin. use '__ALL' for any"
```

Comment:
Allowing `-corsOrigin '*'` in the CLI is dangerour... If the user forget to include `*` as literal, the shell will extand it with the contents of the current directory. @jmcanterafonseca do you see any problem in using an alias ("__ALL" could be changed for a better one if you want)? Of course, this precludes to use literarly __ALL as domain origin is a name weird enough so nobody should be using it as real domain name :)